packages/chain-agnostic-permission/CHANGELOG.md

```diff
@@ -7,6 +7,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Added `getCaipAccountIdsFromCaip25CaveatValue`, `isInternalAccountInPermittedAccountIds`, and `isCaipAccountIdInPermittedAccountIds` account id functions. ([#5609](https://github.com/MetaMask/core/pull/5609))
+- Added `getAllScopesFromCaip25CaveatValue`, `getAllNonWalletNamespacesFromCaip25CaveatValue`, `getAllScopesFromPermission`, `getAllScopesFromCaip25CaveatValue`, and `isNamespaceInScopesObject`
+  scope functions. ([#5609](https://github.com/MetaMask/core/pull/5609))
+- Added `getCaip25CaveatFromPermission` misc functions. ([#5609](https://github.com/MetaMask/core/pull/5609))
+
+### Changed
+
+- **BREAKING:** Renamed `setPermittedAccounts` to `setNonSCACaipAccountIdsInCaip25CaveatValue`. ([#5609](https://github.com/MetaMask/core/pull/5609))
+- **BREAKING:** Renamed `setPermittedChainIds` to `overwriteCaipChainIdsInCaip25CaveatValue`. ([#5609](https://github.com/MetaMask/core/pull/5609))
```

Comment:
needs to be updated

Reply:
fixed here: https://github.com/MetaMask/core/pull/5609/commits/b649dfcc2c36349ab50582aed88d6d15b2d19437